### ISSUE TITLE
Many Blocks can't be waterlogged in acid

### ISSUE THREAD

**Issue by Jasdernn** [AlexModGuy/AlexsCaves#189]
Should it be called Acid-logging? 
Blocks Affected (There could always be more):
Galena Spire, Pointed Dripstone, Cauldrons, bioluminescent torches, sirens, all fences, all walls, all stairs, all slabs, glow lichen, candles, sea pickles, lightning rod, all chests, dragon egg, pots, conduit, brewing stand, lectern, ladders, cakes, grindstone, stonecutter, bell, beholder, Amythyst cluster, baleen covered bone, uranium rods, medium and small tuff smokers, sniffer eggs, signs, lanterns, chains, coral, banners, trapdoors or door (but yet again they aren't waterlog able in the base game so...), moth balls, thin bone block, iron bars, tube worm, magnetic lights, forsaken effigy, thornwood branch, pewen branch, Abyssmarine altar, heart of iron, hologram projector, both azure and scarlet neodymium nodes, mussels, copper valve, and campfires.
TLDR: Basically, all non-full blocks with some exceptions
The exceptions: Rebar, Sulfur Clusters, Armor Stand, Item Frame, Painting, and Composter.
They can't be acid-logged after they are placed either.

Sorry it was so long, just trying to make this as easy for you as possible.

**Comment by AlexModGuy** [comment]
intentional

**Comment by Jasdernn** [comment]
Just saying, they would have looked good in there.
